sdk/state/open.go

```diff
@@ -120,6 +126,11 @@ func (c *Channel) ConfirmOpen(m OpenAgreement) (open OpenAgreement, authorized b
 		return m, authorized, fmt.Errorf("input open agreement details do not match the saved open agreement details")
 	}
 
+	// If the expiry of the agreement is past the max expiry the channel will accept, error.
```

Comment:
❓ If my timezone is your timezone minus 1 hour. And you send me an open agreement, this check would fail right?

🤔 if so I'm wondering how we could resolve. Keep everything in perspective of utc, `time.Now().UTC()`?

Reply:
I don't think we need the UTC here because the time object will contain the local timezone and convert it appropriately in the comparison. However we do need to use it before converting to unix at the other spot because unix time doesn't encode the timezone.

Reply:
🤔 right because the time object contains the local timezone wouldn't the example I gave above stand? eg.
```
Initiator timezone: UTC - 0. time.Now() = 3:00
Responder timezone: UTC - 2. time.Now() = 1:00

Initiator proposes open with expiry at 3:05

Responder calls ConfirmOpen, and 3:05 > 1:05, so this would error.
```
Right? (I might be thinking about it wrong)

Reply:
You are correct except those times all contain time zones so they all compare fine.

```
Initiator timezone: UTC - 0. time.Now() = 3:00 UTC-0
Responder timezone: UTC - 2. time.Now() = 1:00 UTC-2

Initiator proposes open with expiry at 3:05 UTC-0 (1:05 UTC-2)

Responder calls ConfirmOpen, and 3:05 UTC-0 == 1:05 UTC-2, so this would be fine.
```

Reply:
yup you're right, took me a second to wrap my head around 👍